dev_requirements.txt

```diff
@@ -5,58 +5,63 @@
 #    pip-compile --allow-unsafe --no-emit-find-links --no-index --output-file=- -
 #
 appdirs==1.4.3            # via black
-attrs==19.3.0             # via black, flake8-bugbear, pytest
-black==19.10b0
+attrs==19.3.0             # via black, flake8-bugbear, pytest, syrupy
+black==19.10b0            # via syrupy
 bleach==3.1.4             # via readme-renderer
-certifi==2019.11.28       # via requests
+certifi==2020.4.5.1       # via requests
 chardet==3.0.4            # via requests
 click==7.1.1              # via black, pip-tools
-codecov==2.0.22
-colored==1.4.2
+codecov==2.0.22           # via syrupy
+colored==1.4.2            # via syrupy
 configparser==5.0.0       # via py-githooks
-coverage[toml]==5.0.4     # via codecov
+coverage[toml]==5.0.4     # via codecov, syrupy
+deprecated==1.2.8         # via pygithub
 docutils==0.16            # via readme-renderer
 entrypoints==0.3          # via flake8
-flake8-bugbear==20.1.4
-flake8-builtins==1.5.2
-flake8-comprehensions==3.2.2
-flake8-i18n==0.1.0
-flake8==3.7.9             # via flake8-bugbear, flake8-builtins, flake8-comprehensions, flake8-i18n
+flake8-bugbear==20.1.4    # via syrupy
+flake8-builtins==1.5.2    # via syrupy
+flake8-comprehensions==3.2.2  # via syrupy
+flake8-i18n==0.1.0        # via syrupy
+flake8==3.7.9             # via flake8-bugbear, flake8-builtins, flake8-comprehensions, flake8-i18n, syrupy
 idna==2.9                 # via requests
-invoke==1.4.1
-isort==4.3.21
+invoke==1.4.1             # via syrupy
+isort==4.3.21             # via syrupy
 keyring==21.2.0           # via twine
 mccabe==0.6.1             # via flake8
 more-itertools==8.2.0     # via pytest
 mypy-extensions==0.4.3    # via mypy
-mypy==0.770
+mypy==0.770               # via syrupy
 packaging==20.3           # via pytest
 pathspec==0.7.0           # via black
-pip-tools==4.5.1
+pip-tools==4.5.1          # via syrupy
 pkginfo==1.5.0.1          # via twine
 pluggy==0.13.1            # via pytest
-py-githooks==1.1.0
+py-githooks==1.1.0        # via syrupy
 py==1.8.1                 # via pytest
 pycodestyle==2.5.0        # via flake8
 pyflakes==2.1.1           # via flake8
+pygithub==1.47            # via syrupy
 pygments==2.6.1           # via readme-renderer
-pyparsing==2.4.6          # via packaging
-pytest==5.4.1
+pyjwt==1.7.1              # via pygithub
+pyparsing==2.4.7          # via packaging
+pyperf==2.0.0             # via syrupy
+pytest==5.4.1             # via syrupy
```

Comment:
That doesn't seem right? It's not a dependency of syrupy

Reply:
Not sure why it's doing this yet but will fix before merging

Reply:
Technically it is a dependency of syrupy, it weird but it was introduced in https://github.com/jazzband/pip-tools/releases/tag/4.5.0